src/main/java/com/example/universe/simulator/entityservice/controllers/GalaxyController.java

```diff
@@ -35,6 +37,7 @@
 
     @PostMapping("get-list")
     private Callable<Page<GalaxyDto>> getList(@RequestBody Optional<GalaxyFilter> filter, Pageable pageable) {
+        log.info("calling getList with filter [" + filter.orElse(null) + "] and " + pageable);
```

Comment:
use "log.info(String format, Object... arguments)" instead of "log.info(String msg)".
e.g. log.info("calling getList with filter [{}] and {}", filter.orElse(null), pageable);